lib/OpenQA/WebAPI.pm

```diff
@@ -32,17 +33,23 @@ sub log_name { $$ }
 sub startup {
     my $self = shift;
 
-    # Provide help to users early to prevent failing later on
-    # misconfigurations
-    return if $ENV{MOJO_HELP};
-
-    # "templates/webapi" prefix
-    $self->renderer->paths->[0] = path($self->renderer->paths->[0])->child('webapi')->to_string;
-
     # Some plugins are shared between openQA micro services
     push @{$self->plugins->namespaces}, 'OpenQA::Shared::Plugin';
     $self->plugin('SharedHelpers');
 
+    # Provide help to users early to prevent failing later on misconfigurations
+    return try {
+        $self->config->{auth}->{method} = 'Fake';
+        OpenQA::Setup::load_plugins($self);
```

Comment:
Preparing a static config seems wrong... if the point is to load plugins, you would not be getting the correct one(s)? :thinking: Also, we're not loading any plugins here... but maybe I'm unclear what this is helping with.

Reply:
Right, I made it load the actual config. I've also added a note to make it clear why plugins are loaded at this point.